tools/memap.py

```diff
@@ -10,10 +10,11 @@
 import argparse
 from prettytable import PrettyTable
 
-from tools.utils import argparse_filestring_type, \
+from utils import argparse_filestring_type, \
     argparse_lowercase_hyphen_type, argparse_uppercase_type
 
 DEBUG = False
+DETAILED = False
```

Comment:
You could: remove the `@statictmethod` decorator from `path_object_to_module_name` giving you access to `self` (you would also have to add it as the first parameter). Then you could pass `detailed` into the constructor, with a default value of `False`